build/workbox.config.json

```diff
@@ -0,0 +1,15 @@
+{
+    "globDirectory": "./",
+    "globPatterns": [
+        "assets/**/*.{js,svg,png,css}",
+        "dist/**/*.{js,css}",
+        "docs/**/*.{md,html,css,jpg,js}"
+    ],
+    "swSrc": "./sw.js",
+    "swDest": "./_gh_pages/sw.js",
+    "templatedUrls": {
+      "/": ["_gh_pages/docs/index.html"],
+      "docs": ["_gh_pages/docs/index.html"],
+      "docs/4.0/": ["_gh_pages/docs/4.0/index.html"] 
```

Comment:
So 'templatedUrls' is just a way to redirect urls to their relative sources. Since the docs uses /docs/index instead.html of _gh_pages/docs/index.html and _gh_pages/* doesn't actually exist via 'http://getbootstrap.com/gh_pages/*', we're basically saying get the source contents of _gh_pages/docs/index.html and the cache the contents of it as if you're visiting the url '/'.